app/jenkins-for-jira-ui/src/components/ConnectionComplete/CompleteConnection.test.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import {
-	render, screen, fireEvent, act
+	render, screen, act
```

Comment:
I noticed this morning the linter hook is throwing some errors but then continuing to run the tests instead of failing. Will have to have a look at that :susp: